index.html

```diff
@@ -14,249 +14,249 @@
 </head>
 
 <body>
-    <header class="">
-        <div class="nav-bar">
+    <header>
+        <div class="nav-bar flex">
             <div>
                 <div class="topnav">
                     <a class="logo" href="#">My Logo</a>
                     <div class="nav-right ">
-                        <a id="menu-button" class="item-navigation" href="#"><i class="fa fa-bars"></i></a>
-                        <a class="item-navigation hidden" href="#card">Portfolio</a>
-                        <a class="item-navigation hidden" href="#about ">About</a>
-                        <a class="item-navigation hidden" href="#contact ">Contact</a>
+                        <a id="menu-button" class="item-navigation menu-button" href=" #"><i class="fa fa-bars "></i></a>
+                        <a class="item-navigation hidden " href="#home ">Portfolio</a>
+                        <a class="item-navigation hidden " href="#about ">About</a>
+                        <a class="item-navigation hidden " href="#contact ">Contact</a>
                     </div>
                 </div>
             </div>
             <div id="mobile-menu">
-                <a id="close" href="#"><i class="fa fa-close"></i></a>
-                <ul>
-                    <li><a class="mobile-item-navigation" href="#card">Portfolio</a></li>
-                    <li><a class="mobile-item-navigation" href="#about ">About</a></li>
-                    <li><a class="mobile-item-navigation" href="#contact ">Contact</a></li>
+                <a id="closeButton" href="# "><i class="fa fa-close close"></i></a>
+                <ul class="closeItem">
+                    <li><a href="#home ">Portfolio</a></li>
+                    <li><a href="#about ">About</a></li>
+                    <li><a href="#contact ">Contact</a></li>
```

Comment:
- [ ] OPTIONAL: These links are not working on Firefox!.  This is because there is an empty space at the end of each of them.  If you delete the empty space after the end of each of them, they will work (tested on both Chrome and Firefox).  

eg:

Instead of:
```HTML
<a href="#home ">Portfolio</a>
```

Do this:

```HTML
<a href="#home">Portfolio</a> 
```

That way, the links will work on both browsers!. 💪

Reply:
Alright, I will edit it